core/templates/pages/classroom-page/classroom-page.component.html

```diff
@@ -14,16 +14,10 @@ <h2 translate="I18N_CLASSROOM_PAGE_TOPICS_COVERED"></h2>
     <p> <[$ctrl.classroomData.getTopicListIntro()]> </p>
   </div>
   <div class="oppia-topic-summary-tiles">
-    <div ng-repeat="topicSummary in $ctrl.classroomData.getPublicTopicSummaries()" class="oppia-topic-summary-tile protractor-test-topic-summary-tile">
+    <div ng-repeat="topicSummary in $ctrl.classroomData.getTopicSummaries()" class="oppia-topic-summary-tile protractor-test-topic-summary-tile">
       <topic-summary-tile topic-summary="topicSummary"
                           classroom-url-fragment="$ctrl.classroomUrlFragment"
-                          published="true">
-      </topic-summary-tile>
-    </div>
-    <div ng-repeat="topicSummary in $ctrl.classroomData.getPrivateTopicSummaries()" class="oppia-topic-summary-tile protractor-test-topic-summary-tile">
-      <topic-summary-tile topic-summary="topicSummary"
-                          classroom-url-fragment="$ctrl.classroomUrlFragment"
-                          published="false">
+                          published="topicSummary.isPublished">
```

Comment:
If it does have the field, shouldn't this be `isPublished()`?

Reply:
Hm, currently it is a public property in the frontend model. So I've accessed it this way.

https://github.com/oppia/oppia/blob/74379473dd49a2ed7becb7d1fd24ed45253105d5/core/templates/domain/topic/topic-summary.model.ts#L56-L58

I could add a getter if required.

Reply:
Done.